src/main/java/com/google/sps/servlets/EmailServlet.java

```diff
@@ -0,0 +1,168 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.Properties;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet that sends emails to the author and followers of an answered question or a commented
+ * answer.
+ */
+@WebServlet("/email")
+public class EmailServlet extends HttpServlet {
+
+  /**
+   * Sends a welcome email to the user after they register
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get type from query string.
+    String typeOfElement = request.getParameter("type");
+    // Get ID from query string and convert to int.
+    int elementId = Integer.parseInt(request.getParameter("elementId"));
```

Comment:
I'm kind of confused what elementId and typeOfElement are. Are these User IDs? Can you rename these variables to be clearer or add comments explaining?

Reply:
Ack! Let me know if further explanation is required.